### PR TITLE
Build a dev docker image

### PR DESCRIPTION
**Pull request by bzwei**
To build a docker image:
```
docker build -t service_catalog --file=DevDockerfile .
```

To run the docker container:
```
docker run -p 5000:5000 --name=approval_service -e DATABASE_URL="postgres://myuser:mypass@0.0.0.0/service_approval_development" -d approval_service
```
In the above command line you need to feed in the correct setting to connect to a database for approval service. Database username and password are optional if your testing Postgresql does not require them.

### CODE REVIEW

**Comment by bzwei** [comment]
@syncrou @gmcculloug @hsong-rh

File: config/database.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ default: &default
   password: <%= ENV['SERVICE_APPROVAL_DATABASE_PASSWORD'] %>
   wait_timeout: 5
   min_messages: warning
-  host: localhost
+  host: <%= ENV['SERVICE_APPROVAL_DATABASE_HOST'] %>
```
Comment (syncrou) [inline config/database.yml]: `database.yml` should be replaced by a sample [database.dev.yml](https://github.com/ManageIQ/service_portal-api/blob/master/config/database.dev.yml) so the real database.yml doesn't create dirty commits.

Comment (syncrou) [reply config/database.yml]: You can then replace the `<%= ENV['SERVICE_APPROVAL_DATABASE_HOST'] %>` environment variable in the `docker run` command with `DATABASE_URL`.

Comment (bzwei) [reply config/database.yml]: my goal is to have a `database.yml` that does not need modification.

File: config/ssl/localhost.key [hunk 2]
```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```
Comment (carbonin) [inline config/ssl/localhost.key]: Don't commit private keys.

File: DevDockerfile [hunk 3]
```diff
@@ -1,21 +1,33 @@
...
-
-WORKDIR $RAILS_ROOT
-
-COPY Gemfile Gemfile
-
-RUN gem install bundler
-
-RUN bundle install
-
+FROM manageiq/ruby:latest
```
Comment (carbonin) [inline DevDockerfile]: This looks almost exactly like `Dockerfile` in this repo. If the image is going to be nearly identical, why have two?

File: config/ssl/localhost.key (hunk 2 above)
Comment (carbonin) [reply config/ssl/localhost.key]: If you need to generate a cert at build or runtime [here](https://github.com/ManageIQ/manageiq-appliance/blob/master/LINK/usr/bin/generate_miq_server_cert.sh) is an example of how we do it for manageiq

**Comment by bzwei** [comment]
no longer needed

**Pull request closed without merging**